### ISSUE TITLE
Memory consumption in IE 11

### ISSUE THREAD

**Issue by may16197** [exupero/saveSvgAsPng#131]
Hi,
I am using IE11 and after 3 to 4 downloads, browser hangs.

regds,
DD

**Comment by exupero** [comment]
Are these very large SVGs?

Are there any errors or other messages in the developer console?

**Comment by may16197** [comment]
even if i have 50 to 60 nodes, it hangs after 4 to 5 downaloads.. As such there are no error/any messages. But in the task manager, i could see the growth in memory consumption.

**Comment by exupero** [comment]
Does the number of downloads before hanging change at all as you reduce the complexity of the SVG?

Do you see the same behavior in any other browsers?

Does your SVG contain any images that have to be inlined?

Do you see the same behavior if you exclude all CSS?

A bug in the image or CSS inlining node is the only thing in the script I can think of that would consume a noticeable amount of memory.

**Comment by may16197** [comment]
yes.. if I have less number of nodes, browser hangs after 7 to 8 downloads. and sometimes if we reload the browser , that doesnt seems to help as well..
we do have images for nodes and links. I have not seen this behaviour in firefox. it works fine there. Only IE has this issue.

**Comment by FlorianLeitgeb** [comment]
This is not related to the topic, but could you maybe post your code? Because I am not able to get this running at all in IE11. I know the thing with the canvg, but I am doing something wrong.

**Comment by may16197** [comment]
Hey..sry!! I left tht company around an year ago..with canvg too, u HV to
make some more changes..I am not able to recall those changes.. Really sry
for not able to help u on this..

Cheers,
D
